package.xml

```diff
@@ -19,6 +19,7 @@
     <depend condition="$ROS_VERSION == 1">rospack</depend>
     <depend condition="$ROS_VERSION == 2">ament_index_cpp</depend>
     <depend condition="$ROS_VERSION == 2">rclcpp</depend>
+    <depend>libwebsocketpp-dev</depend>
```

Comment:
What makes this a depend vs build_depend

Reply:
This is a legit question. I don't know why I changed it, it should have remained a build dependency since it is only required during build.

Reply:
It's a `<build_depend>` because websocketpp is a header-only library. If it had a shared runtime library there would be a package called `websocketpp` that would be a `<depend>` while libwebsoscketpp-dev remained a `<build_depend>`

Reply:
-> #27